imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -0,0 +1,50 @@
+// +build linux
+
+package imageboot
+
+import (
+	"flag"
+	"os"
+	"syscall"
+	"testing"
+)
+
+var (
+	runtest = flag.Bool("runtest", false, "really run the test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if *runtest {
+		os.Exit(m.Run())
+	} else {
+		os.Exit(0)
+	}
+}
+
+func TestGuestBoot(t *testing.T) {
+	err := syscall.Uname(&syscall.Utsname{})
```

Comment:
i think simply reaching this function fulfills the expectation and we can omit the syscall. then it will also be a good test for windows.

Reply:
The BCT test run a command `uname` as a signal for success boot. so I find an equivalent syscall like that cmd. But I agree that the test function itself is fulfills the expectation.